app/views/mailers/notify_mailer/unread_notifications_email.text.erb

```diff
@@ -1 +1 @@
-Visit https://dev.to/notifications to read all of your notifications
+Visit <%= app_url("/notifications") %> to read all of your notifications
```

Comment:
`app_url(notifications_path)`

Reply:
@rhymes this doesn't work notifications_path in the view is undefined.